packages/core/src/elements/ResponsiveElement.ts

```diff
@@ -30,13 +30,22 @@ export type ResizeEvent = CustomEvent<{
  * @returns {void}
  */
 const triggerResize = (entry: ResizeObserverEntry): void => {
-  const { inlineSize: width, blockSize: height } = entry.borderBoxSize[0];
+  let width: number;
+  let height: number;
+  if (entry.borderBoxSize?.length > 0) {
```

Comment:
could you add above this line? This should improve code coverage report from SonarCloud.
`/* istanbul ignore else */`

Reply:
Done